comment/static/js/comment.js

```diff
@@ -7,6 +7,16 @@ document.addEventListener('DOMContentLoaded', () => {
     let csrfToken = window.CSRF_TOKEN;
     document.getElementsByClassName(".js-comment-input").value = '';
 
+    let removeTargetElement = e => {
```

Comment:
currently, if you click on the permalink icon on any of the comments, the respective html element gets highlighted. But this element remains highlighted even on subsequent clicks, because the address bar on the top has the element's id. This function firstly removes the id(anything after the `#` symbol including the `#`). After remove the id from the address bar the browser tends to scroll back to the top of the window, this function also retains the window to the current scrolled height.